presentation/common/src/presentation-common/PropertyFormatter.ts

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Core
+ */
+
+import { assert } from "@itwin/core-bentley";
+import { Format, FormatProps, FormatterSpec, ParserSpec, UnitsProvider } from "@itwin/core-quantity";
+import { Content } from "./content/Content";
+import { Field, PropertiesField } from "./content/Fields";
+import { DisplayValue, NestedContentValue, Value } from "./content/Value";
+import { KindOfQuantityInfo, PropertyInfo } from "./EC";
+import { ValuesDictionary } from "./Utils";
+
+/** @alpha */
+export interface FormatOptions {
+  name: string;
```

Comment:
not clear whose name this is - maybe rename to `koqName` or something like that?

Reply:
This is the name of `Format` and `FormatterSpec` not `KOQ`. Both requires name to be provided so I just decided to used name of `KOQ` for that.

Reply:
Removed name as it is not used by `Format` or `FormatterSpec`